crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -0,0 +1,1008 @@
+        billing_country: billing_address.get_country()?.to_owned(),
+        billing_street1: billing_address.get_line1()?.to_owned(),
+        billing_postcode: billing_address.get_zip()?.to_owned(),
+    })
+}
+
+fn get_card_request_data(
+    item: &types::PaymentsAuthorizeRouterData,
+    browser_info: &BrowserInformation,
+    params: TrustpayMandatoryParams,
+    amount: String,
+    ccard: &api_models::payments::Card,
+    return_url: String,
+) -> TrustpayPaymentsRequest {
+    TrustpayPaymentsRequest::CardsPaymentRequest(Box::new(PaymentRequestCards {
+        amount,
+        currency: item.request.currency.to_string(),
+        pan: ccard.card_number.clone(),
+        cvv: ccard.card_cvc.clone(),
+        expiry_date: ccard.get_card_expiry_month_year_2_digit_with_delimiter("/".to_owned()),
+        cardholder: ccard.card_holder_name.clone(),
+        reference: item.payment_id.clone(),
+        redirect_url: return_url,
+        billing_city: params.billing_city,
+        billing_country: params.billing_country,
+        billing_street1: params.billing_street1,
+        billing_postcode: params.billing_postcode,
+        customer_email: item.request.email.clone(),
+        customer_ip_address: browser_info.ip_address,
+        browser_accept_header: browser_info.accept_header.clone(),
+        browser_language: browser_info.language.clone(),
+        browser_screen_height: browser_info.screen_height.clone().to_string(),
+        browser_screen_width: browser_info.screen_width.clone().to_string(),
+        browser_timezone: browser_info.time_zone.clone().to_string(),
+        browser_user_agent: browser_info.user_agent.clone(),
+        browser_java_enabled: browser_info.java_enabled.clone().to_string(),
+        browser_java_script_enabled: browser_info.java_script_enabled.clone().to_string(),
+        browser_screen_color_depth: browser_info.color_depth.clone().to_string(),
+        browser_challenge_window: "1".to_string(),
+        payment_action: None,
+        payment_type: "Plain".to_string(),
+    }))
+}
+
+fn get_bank_redirection_request_data(
+    item: &types::PaymentsAuthorizeRouterData,
+    bank_redirection_data: &BankRedirectData,
+    amount: String,
+    return_url: String,
+    auth: TrustpayAuthType,
+) -> TrustpayPaymentsRequest {
+    TrustpayPaymentsRequest::BankRedirectPaymentRequest(PaymentRequestBankRedirect {
+        payment_method: get_trustpay_payment_method(bank_redirection_data),
+        merchant_identification: MerchantIdentification {
+            project_id: auth.project_id,
+        },
+        payment_information: BankPaymentInformation {
+            amount: Amount {
+                amount,
+                currency: item.request.currency.to_string(),
+            },
+            references: References {
+                merchant_reference: item.payment_id.clone(),
+            },
+        },
+        callback_urls: CallbackURLs {
+            success: format!("{}?status=SuccessOk", return_url),
```

Comment:
Use the same string as used in `Status`

Reply:
In cards they are returning with status=SuccessOk, so have added same here to handle both the cases